sideboard/lib/sa/__init__.py

```diff
@@ -261,7 +282,7 @@ def resolve_model(cls, name):
                     return subclasses[singular]
 
             if name.lower().endswith('ies'):
-                singular = name[:-3] + 'sy'  # TODO: sy looks like a typo, and we need to either make this better or get rid of it
+                singular = name[:-3] + 'y'
```

Comment:
Agree that ``TODO`` in code is a smell and things should be ticketed.

This is technically a backwards-incompatible change, but in this case I think it's okay since AFAIK there are no Sideboard plugins in existence which rely on this behavior.